assets/js/vue/components/stats/RepositorySummary.vue

```diff
@@ -93,4 +93,15 @@ export default {
     font-size: 18px;
     text-transform: uppercase;
 }
+
+/* Center icons above counts/labels */
+#program-resources .col-sm {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#program-resources .col-sm img.inline-block {
+    margin-bottom: 6px;
+}
```

Comment:
Maybe we can give this section a class name, classes tend to stick around longer, and id may be removed because it seems un-needed.